### PR TITLE
Add testimonials section with carousel

### PR DESCRIPTION
**Pull request by jasondippel**
# What Changed
- Adding testimonials section
- Created Testimonials components (basically a carousel specific to testimonials right now; could generalize later if needed)
- Adding in @glidejs/glide lib to help create carousel

## Screenshots
Kapture 2021-01-28 at 16 42 36

### CODE REVIEW

File: components/testimonials/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import { useEffect } from 'react'
+import styled from 'styled-components'
+import Slide from './slide'
+import '@glidejs/glide/dist/css/glide.core.min.css'
+import '@glidejs/glide/dist/css/glide.theme.min.css'
+
+const DOT_SIZE = rem('18px')
+const DOT_SIZE_MOBILE = rem('24px')
+
+const Root = styled.div`
```
Comment (julie-huang) [inline components/testimonials/index.js:12]: Maybe we want to move styles to `components/testimonials/styles.js` 

What you guys think? @superkhau @timothy-wan

File: components/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 export { default as ArticlePreview } from './article-preview'
 export { default as Button } from './button'
+export { default as Testimonials } from './testimonials'
```
Comment (julie-huang) [inline components/index.js]: super nit picky, but sort please 🙏🏼

**Review by julie-huang — CHANGES_REQUESTED** [review]
Minor stuff, will approve after getting some second opinion after styles convention

File: components/testimonials/index.js (hunk 1 above)
Comment (timothy-wan) [reply components/testimonials/index.js:12]: Well styled components are laid out like this anyway. I'm good leaving like this

File: components/index.js (hunk 2 above)
Comment (jasondippel) [reply components/index.js]: fixed :)

**Review by superkhau — APPROVED** [review]

**Review by julie-huang — APPROVED** [review]

**Pull request merged**